engine/engine_test.go

```diff
@@ -2240,6 +2240,13 @@ avg by (storage_info) (
 			    http_requests_total{"label:name"="labelvalue"} 1+2x40`,
 			query: `rate(http_requests_total{"label:name"="labelvalue"}[20s:10s] offset 20s)`,
 		},
+		{
+			name: "native histogram unsupported functions",
```

Comment:
Can we be more specific about the function?
Also let's add test cases for other functions we changed in this PR?